src/services/ExperienceService.ts

```
@@ -0,0 +1,83 @@
+      if (!res.ok) {
+        const errorData = await res.json().catch(() => null);
+        throw new Error(errorData?.message || `Failed to update experience: ${res.status}`);
+      }
+      
+      return res.json();
+    } catch (error) {
+      console.error("Error updating experience:", error);
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)